h2o-extensions/xgboost/testMultiNode.sh

```diff
@@ -100,9 +100,9 @@ echo $DOONLY > $OUTDIR/tests.doonly.txt
 # Launch 3 helper JVMs.  All output redir'd at the OS level to sandbox files.
 CLUSTER_NAME=junit_cluster_$$
 CLUSTER_BASEPORT=44000
-$JVM water.H2O -name $CLUSTER_NAME -ip $H2O_NODE_IP -baseport $CLUSTER_BASEPORT -ga_opt_out $SSL 1> $OUTDIR/out.1 2>&1 & PID_1=$!
-$JVM water.H2O -name $CLUSTER_NAME -ip $H2O_NODE_IP -baseport $CLUSTER_BASEPORT -ga_opt_out $SSL 1> $OUTDIR/out.2 2>&1 & PID_2=$!
-$JVM water.H2O -name $CLUSTER_NAME -ip $H2O_NODE_IP -baseport $CLUSTER_BASEPORT -ga_opt_out $SSL 1> $OUTDIR/out.3 2>&1 & PID_3=$!
+$JVM water.H2OApp -name $CLUSTER_NAME -ip $H2O_NODE_IP -baseport $CLUSTER_BASEPORT -ga_opt_out $SSL 1> $OUTDIR/out.1 2>&1 & PID_1=$!
```

Comment:
Since REST API is required for multinode XGBoost I had to switch this to H2OApp to enable it, this is non standard @Pscheidl @michalkurka do you see a better solution?

Reply:
I don't see any problem in what you did whatsoever.

Reply:
@honzasterba I don't like the dependency on h2o-app; imho it would be better to avoid that because that brings in other dependencies

IMHO it would be better to use `H2OTestNodeStarter` instead of H2OApp, it will do the same thing